plasmapy/examples/basic_simulation.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import astropy.units as u
+import matplotlib.pyplot as plt
+from pathlib import Path
+from plasmapy import Plasma
+
+
+def gaussian(x, mean=0.0, std=1.0, amp=1.0):
+    """Simple function to return a Gaussian distribution"""
+    if isinstance(x, list):
+        x = np.array(x)
+    power = -((x - mean) ** 2.0) / (2.0 * (std ** 2.0))
+    f = amp * np.exp(power)
+    if amp == 1:
+        f = f / max(f)
```

Comment:
It seems better to use `np.max` here, it's better optimized for arrays. I'm not particularly sure why do we need this `if` statement, though.